kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -20,6 +20,11 @@
 import com.amazonaws.auth.DefaultAWSCredentialsProviderChain;
 import com.amazonaws.regions.RegionUtils;
 import com.amazonaws.regions.Regions;
+import io.confluent.connect.storage.StorageSinkConnectorConfig;
```

Comment:
Yes, it's not ideal that you have to diverge from the default ordering of imports. But it makes sense to have it that way. 

To give you a hint, here's what the rules are regarding imports in `codeStyleSettings.xml` when using IntelliJ: 

```
        <option name="IMPORT_LAYOUT_TABLE">
          <value>
            <package name="" withSubpackages="true" static="false" />
            <emptyLine />
            <package name="javax" withSubpackages="true" static="false" />
            <package name="java" withSubpackages="true" static="false" />
            <emptyLine />
            <package name="io.confluent" withSubpackages="true" static="false" />
            <emptyLine />
            <package name="" withSubpackages="true" static="true" />
          </value>
        </option>
```